mail/api_test.py

```diff
@@ -122,16 +131,42 @@ class FinancialAidMailAPITests(TestCase):
     """
     Tests for the Mailgun client class for financial aid
     """
+    @classmethod
+    def setUpTestData(cls):
+        with mute_signals(post_save):
+            cls.staff_user_profile = ProfileFactory.create()
+            cls.financial_aid = FinancialAidFactory.create()
+
     @override_settings(MAILGUN_FROM_EMAIL='mailgun_from_email@example.com')
     def test_financial_aid_email(self, mock_post):
         """
         Test that MailgunClient.send_financial_aid_email() sends an individual message
         """
-        MailgunClient.send_financial_aid_email('email subject', 'email body', 'a@example.com')
+        mock_post.return_value = Mock(
+            spec=Response,
+            status_code=HTTP_200_OK,
+            json=mocked_json()
+        )
+        assert FinancialAidEmailAudit.objects.count() == 0
+        MailgunClient.send_financial_aid_email(
+            self.staff_user_profile.user,
+            self.financial_aid,
+            'email subject',
+            'email body'
+        )
+        # Check method call
         assert mock_post.called
         called_args, called_kwargs = mock_post.call_args
         assert list(called_args)[0] == '{}/{}'.format(settings.MAILGUN_URL, 'messages')
         assert called_kwargs['auth'] == ('api', settings.MAILGUN_KEY)
         assert called_kwargs['data']['text'].startswith('email body')
         assert called_kwargs['data']['subject'] == 'email subject'
-        assert called_kwargs['data']['to'] == ['a@example.com']
+        assert called_kwargs['data']['to'] == [self.financial_aid.user.email]
+        # Check audit creation
+        assert FinancialAidEmailAudit.objects.count() == 1
+        audit = FinancialAidEmailAudit.objects.first()
+        assert audit.acting_user == self.staff_user_profile.user
+        assert audit.financial_aid == self.financial_aid
+        assert audit.to_email == self.financial_aid.user.email
+        assert audit.email_subject.startswith('email subject')
```

Comment:
If we don't need `startswith` here and above can you change to a direct comparison?